### PR TITLE
Fixed Dockerize lncli failure and docs

### PR DESCRIPTION
**Pull request by patwhite**
In reference to #295, fixed small documentation issue in "Step by Step payment guide" and modified docker script to not specify a data directory (which was preventing lncli form running without additional command line parameters, and breaking the documentation).

I'd be worried moving the data direction may not be backwards compatible, but I'm not sure if people map it or not.

cc @halseth

### CODE REVIEW

File: docker/lnd/start-lnd.sh [hunk 1]
```diff
@@ -46,7 +46,6 @@ NETWORK=$(set_default "$NETWORK" "simnet")
 CHAIN=$(set_default "$CHAIN" "bitcoin")
 
 lnd \
-    --datadir="/data" \
```
Comment (Roasbeef) [inline docker/lnd/start-lnd.sh]: I'd also argue that the `logdir` itself doesn't need to be modified either.

**Comment by patwhite** [comment]
@Roasbeef do you squash and merging, or need me to make a cleaner PR?

**Comment by halseth** [comment]
@patwhite Would be nice if you could squash into one commit with a nice description (check contribution guidelines) and rebase on master :)

File: docker/lnd/start-lnd.sh (hunk 1 above)
Comment (halseth) [reply docker/lnd/start-lnd.sh]: Yeah, probably remove the logdir flag also

**Pull request merged**